packages/aws-cdk-lib/aws-rds/lib/instance-engine.ts

```diff
@@ -1841,67 +2249,159 @@ class OracleEeCdbInstanceEngine extends OracleInstanceEngineBase {
-  /** Version "11.00.5058.0.v1". */
+  /**
+   * Version "11.00.5058.0.v1".
+   * @deprecated SQL Server 11.00.5058.0.v1 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_11_00_5058_0_V1 = SqlServerEngineVersion.of('11.00.5058.0.v1', '11.00');
-  /** Version "11.00.6020.0.v1". */
+  /**
+   * Version "11.00.6020.0.v1".
+   * @deprecated SQL Server 11.00.6020.0.v1 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_11_00_6020_0_V1 = SqlServerEngineVersion.of('11.00.6020.0.v1', '11.00');
-  /** Version "11.00.6594.0.v1". */
+  /**
+   * Version "11.00.6594.0.v1".
+   * @deprecated SQL Server 11.00.6594.0.v1 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_11_00_6594_0_V1 = SqlServerEngineVersion.of('11.00.6594.0.v1', '11.00');
-  /** Version "11.00.7462.6.v1". */
+  /**
+   * Version "11.00.7462.6.v1".
+   * @deprecated SQL Server 11.00.7462.6.v1 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_11_00_7462_6_V1 = SqlServerEngineVersion.of('11.00.7462.6.v1', '11.00');
-  /** Version "11.00.7493.4.v1". */
+  /**
+   * Version "11.00.7493.4.v1".
+   * @deprecated SQL Server 11.00.7493.4.v1 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_11_00_7493_4_V1 = SqlServerEngineVersion.of('11.00.7493.4.v1', '11.00');
 
-  /** Version "12.00" (only a major version, without a specific minor version). */
+  /**
+   * Version "12.00" (only a major version, without a specific minor version).
+   * @deprecated SQL Server 12.00 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_12 = SqlServerEngineVersion.of('12.00', '12.00');
-  /** Version "12.00.5000.0.v1". */
+  /**
+   * Version "12.00.4422.0.v1"
+   * @deprecated SQL Server 12.00.4422.0.v1 is no longer supported by Amazon RDS.
+   */
+  public static readonly VER_12_00_4422_0_V1 = SqlServerEngineVersion.of('12.00.4422.0.v1', '12.00');
+  /**
+   * Version "12.00.5000.0.v1".
+   * @deprecated SQL Server 12.00.5000.0.v1 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_12_00_5000_0_V1 = SqlServerEngineVersion.of('12.00.5000.0.v1', '12.00');
-  /** Version "12.00.5546.0.v1". */
+  /**
+   * Version "12.00.5546.0.v1".
+   * @deprecated SQL Server 12.00.5546.0.v1 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_12_00_5546_0_V1 = SqlServerEngineVersion.of('12.00.5546.0.v1', '12.00');
-  /** Version "12.00.5571.0.v1". */
+  /**
+   * Version "12.00.5571.0.v1".
+   * @deprecated SQL Server 12.00.5571.0.v1 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_12_00_5571_0_V1 = SqlServerEngineVersion.of('12.00.5571.0.v1', '12.00');
-  /** Version "12.00.6293.0.v1". */
+  /**
+   * Version "12.00.6293.0.v1".
+   * @deprecated SQL Server 12.00.6293.0.v1 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_12_00_6293_0_V1 = SqlServerEngineVersion.of('12.00.6293.0.v1', '12.00');
```

Comment:
`12.00.6293.0.v1` and above seem still supported from [docs](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_SQLServer.html#SQLServer.Concepts.General.VersionSupport).

Reply:
Again, not listed by the SDK:

```sh
$ aws rds describe-db-engine-versions --engine sqlserver-se --region us-east-1 | jq -r '.DBEngineVersions[] | select(.EngineVersion | startswith("12."))'
$ aws rds describe-db-engine-versions --engine sqlserver-ee --region us-east-1 | jq -r '.DBEngineVersions[] | select(.EngineVersion | startswith("12."))'
$ aws rds describe-db-engine-versions --engine sqlserver-ex --region us-east-1 | jq -r '.DBEngineVersions[] | select(.EngineVersion | startswith("12."))'
$ aws rds describe-db-engine-versions --engine sqlserver-web --region us-east-1 | jq -r '.DBEngineVersions[] | select(.EngineVersion | startswith("12."))'
```